python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -90,6 +91,36 @@ def closeserver():
 
             atexit.register(closeserver)
 
+        def get_base_url(self):
+            env = EnvironmentDetector()
+            detected_envs = env.detect()
+            in_cloud_env = is_cloud_env(detected_envs)
+            # First handle known cloud environments
+            nbvm_file_path = "/mnt/azmnt/.nbvm"
+            if not (os.path.exists(nbvm_file_path) and os.path.isfile(nbvm_file_path)):
```

Comment:
I think this can be broken into several functions to make it more general, eg one function to determine if we are in a notebook vm and another function to return the URL for notebook VM, which might make it easier to maintain in the future and will make this look more general (since this repo isn't azure specific)

Reply:
feel free to make those changes later though as we have a regular release of interpret-community today I believe

Reply:
exactly, there is a epic for next semester to extract all this environment detection and parameterization into a package that we and interpret would use (and fairlearn)